bin/prettier.js

```diff
@@ -280,7 +280,7 @@ if (stdin) {
   });
 } else {
   eachFilename(filepatterns, filename => {
-    if (write || argv["debug-check"]) {
```

Comment:
The filename still gets printed out [here](https://github.com/prettier/prettier/pull/2013/files#diff-970aecfd974e4b7ebfc40bf6329bb7b2R362). I added a test ensuring this: https://github.com/prettier/prettier/pull/2013/commits/2b4c09ded811544e3b9099b515d605b41b2760ce